neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -124,9 +125,11 @@ def __init__(
         self.proxies = proxies
         self.missing_features = []
 
-        self._http_client, self._client_config = create_http_client_with_auth(
+        http_client, client_config = create_http_client_with_auth(
```

Comment:
Let's try the syntax with `#type` (it works locally in my PyCharm):
```suggestion+3
        self._http_client, self._client_config = create_http_client_with_auth(
        ...
        )  # type: (RequestsClient, ClientConfig)
```

Reply:
nice, thanks!